src/components/EventCard.vue

```diff
@@ -1,133 +1,48 @@
 <script setup lang="ts">
-import { type EventMessageName } from '@/i18n/types'
+import type { Event } from '@/i18n/events'
 
-const props = defineProps<{
-  event: EventMessageName
+defineProps<{
+  event: Event
+  featured?: boolean
 }>()
 </script>
 
 <template>
-  <div class="card flex flex-col mb-3 w-[380px] rounded-md overflow-hidden shadow-lg md:flex-row md:h-[260px] md:w-[700px]">
-    <div class="relative h-52 overflow-hidden md:basis-2/5 md:h-auto">
-      <div
-        :data-test="`event-${props.event}-photo`"
-        class="card-image absolute inset-0 bg-center bg-cover"
-        :style="`background-image: url( ${$t(`events.${props.event}.image`)} );`"
-      />
-    </div>
-    <div class="basis-3/5 w-full h-full z-1">
-      <div class="description flex flex-col justify-between items-start h-full p-3 space-y-4 relative lg:space-y-2">
-        <div>
-          <h1
-            class="head-4 mb-2 font-700"
-            :data-test="`event-${props.event}-title`"
-          >
-            {{ $t(`events.${props.event}.title`) }}
-          </h1>
-          <span
-            class="poppins"
-            :data-test="`event-${props.event}-date`"
-          >
-            {{ $t(`events.${props.event}.date`) }} | {{ $t(`events.${props.event}.location`) }}
-          </span>
-        </div>
-        <p
-          :data-test="`event-${props.event}-subtitle`"
-          class="font-bold"
-        >
-          {{ $t(`events.${props.event}.subtitle`) }}
-        </p>
-        <router-link
-          :data-test="`event-${props.event}-read-more`"
-          :to="{ name: 'EventView', params: { event } }"
-          class="read-more p-2 rounded mt-1 self-end relative"
-        >
-          {{ $t(`message.common.read-more`) }}
-        </router-link>
+  <article
+    class="card rounded-3 border border-slate-200/30 bg-dark-bg-secondary"
```

Comment:
Could we have:

`class="card rounded-3 border border-slate-200/30 bg-light-bg-secondary dark:bg-dark-bg-secondary"`

This will handle theme light/dark correctly